### ISSUE TITLE
Can I touch (on the mirror) to interact with my android device ?

### ISSUE THREAD

**Issue by korrio** [HannahMitt/HomeMirror#1]

**Comment by LngDngDnkyKng** [comment]
No, the device is still the contact point. The mirror will not allow touch. You can use a Bluetooth pointing device (mouse/trackball) to interact with the device (not sure if/how well you would see it though).

**Comment by aequitas** [comment]
Maybe removing the glass front of the device and sticking the LCD directly to the mirror would work.

**Comment by davidgoering** [comment]
Hey, I was wondering the very same thing.... here were my ideas:
A tablet with a resistive touch screen would work (not very common). Other than that maybe if the "two-way mirror acrylic sheets" are thin enough it could work, but fairly unlikely. 
Another possible solution could be to use the camera for gesture control or possibly voice commands of some kind, but that may be a bit far fetched for the early stages? :) 
Just my 2 cents.

**Comment by mLuby** [comment]
You might be able to use the device's accelerometer to calculate roughly where the touch occurred.

**Comment by Sanfam** [comment]
Alternatively, some mirrors of this type use a metallic coating on the front layer.  You could make the entire screen a single capacitive button using an MPR121, out frame It and embed a few cap. sensor pads in the frame.

**Comment by philwolstenholme** [comment]
You could use the Android cast screen feature or an app like https://play.google.com/store/apps/details?id=com.koushikdutta.mirror to mirror a device's display elsewhere. This would make it easier to use a Bluetooth pointer.

**Comment by Fleker** [comment]
I think some sort of gesture control or voice would work best for a wall mounted device.

**Comment by JokerQyou** [comment]
Forgive my off-topic discussion, but I thought it was light projected mirror when I first saw the pictures... And I thought you can interact with it through leap motion or something.
